src/lazy.jl

```diff
@@ -430,6 +431,7 @@ function Base.convert(::Type{String}, x::PtrString)
     return unsafe_string(x.ptr, x.len)
 end
 
+# TODO: This ccall uses a private C function.
 Base.convert(::Type{Symbol}, x::PtrString) = ccall(:jl_symbol_n, Ref{Symbol}, (Ptr{UInt8}, Int), x.ptr, x.len)
```

Comment:
Why is this even a convert method? You cannot convert normal strings to Symbols...